Week2/homework/Exercise3.js

```diff
@@ -0,0 +1,16 @@
+//Exercise3.js
+
+'use strict';
+
+const recipe = {
+    MealName: 'Tomato and Egg Stir Fry',
+    Serves: 2,
+    Ingredients:[ '2 eggs', '2 tomatoes', '1/2 tsp. salt', '1/4 cup oil', '1/8 tsp. sugar','1 scallion'],
+};
+
+let entries = Object.entries(recipe);
+
+for(entries of entries){
+    console.log(entries)
+
```

Comment:
Ah this works nicely!

I would recommend to use a different variable name inside the loop, to avoid confusion in the future (and don't forget the `;`.

So it could then be something like this:
```
for(entry of entries){
    console.log(entry);
}
```

but this works fine, so well done!

Reply:
Thank you! Now it's revised. Just a little confused with the entries thing.